Makefile

```diff
@@ -58,6 +60,10 @@ gen:
 		--git-repo-id ${GIT_REPO} \
 		-o /local/${PACKAGE_PREFIX}/${PACKAGE_MAJOR} \
 		-i /local/${SPEC_PATCHED_FILE}
+	# generated tests have a broken import
+	find ${PACKAGE_PREFIX}/${PACKAGE_MAJOR}/test -type f -exec sed -i '' 's_"github.com\/equinix-labs\/metal-go"_"github.com\/equinix-labs\/metal-go\/metal\/v1"_g' {} \;
+    # generated code is missing some types; hack 'em in
+	cat missing_types.go.part >> metal/v1/utils.go
```

Comment:
The Go generator fails to generate a couple types; for now I put them in `missing_types.go.part` to be injected into the `utils.go` file, but we'll need to dig into why they're not generated in the first place.  Based on the names I suspect one or both of the missing types are better represented by built-in Go types, and the generator needs to be updated to take that into account.